roles/json-filter/tasks/json-filter.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- name: json-filter | Clone repo. branch:'{{ json_filter_git_branch }}'
+  ansible.builtin.git:
+    repo: https://github.com/Kv-126-DevOps/json-filter.git
+    dest: '{{ json_filter_workdir }}'
+    version: '{{ json_filter_git_branch }}'
+
+- name: json-filter | Install python packeges into virtualenv
+  pip:
```

Comment:
It is better to use `requirements.txt` file to install all dependencies

Reply:
for requirements.txt we should copy this file to host 
if we need that for just one pkg then ok
i will do it